src/pages/CategoriesPage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- I would advise you to solve the Linter manually. Do not disable linters because it contributes to clean code, and avoid bugs in your code.

Reply:
Thanks for the insight fixed.